slide46/index.html

```diff
@@ -1,87 +1,60 @@
+    <script type="text/javascript" src="https://ajax.googleapis.com/ajax/libs/jqueryui/1.12.1/jquery-ui.min.js"></script>
+
+    <!-- iframe resizer -->
+    <script type="text/javascript" src="js/iframeResizer.min.js"></script>
     <script type="text/javascript" src="js/iframeResizer.contentWindow.min.js"></script>
 
-    <script type="text/javascript" src="DD_slide46.js"></script>
-    <div id="masterContainer" class="masterContainer" data-viewportNarrow="false">
-    <div class="question-container" >
-        <img width="800" style= "z-index:-1" src="slide_46_matching_games.jpg">
-        <button id="submit" style = "float : right; ">Submit</button>
-        <div class="question" style="position: absolute;top: 50px;left: 10px;width: 115px;height: 25px" qid="1">1</div>
-        <div class="question" style="position: absolute;top: 105px;left: 10px;width: 115px; height: 25px" qid="2">2</div>
-        <div class="question" style="position: absolute;top: 155px;left: 10px;width: 115px;height: 25px" qid="3">3</div>
-        <div class="question" style="position: absolute;top: 190px;left: 10px;width: 115px;height: 25px" qid="4">4</div>
-        <div class="question" style="position: absolute;top: 255px;left: 10px;width: 115px; height:25px" qid="5">5</div>
-        <div class="question" style="position: absolute;top: 55px;left: 555px;width: 115px; height:25px" qid="6">6</div>
-        <div class="question" style="position: absolute;top: 105px;left: 555px;width: 115px; height:25px" qid="7">7</div>
-        <div class="question" style="position: absolute;top: 140px;left: 555px;width: 115px; height:25px" qid="8">8</div>
-        <div class="question" style="position: absolute;top: 185px;left: 555px;width: 115px; height:25px" qid="9">9</div>
-        <div class="question" style="position: absolute;top: 215px;left: 555px;width: 115px; height:25px" qid="10">10</div>
-        <div class="question" style="position: absolute;top: 250px;left: 555px;width: 115px; height:25px" qid="11">11</div>
-    </div>
-    <div class= "answer-container">
-      <div class="answer" id="1" >Golgi apparatus</div>
-      <div class="answer" id="2" >Cell membrane</div>
-      <div class="answer" id="3" >Chloroplasts</div>
-      <div class="answer" id="4" >Vacuole</div>
-      <div class="answer" id="5" >Cell wall</div>
-      <div class="answer" id="11" >Mitochondria</div>
-      <div class="answer" id="10" >Cytosol</div>
-      <div class="answer" id="9" >Smooth ER</div>
-      <div class="answer" id="8" >Nucleolus</div>
-      <div class="answer" id="7" >Nucleus</div>
-      <div class="answer" id="6" >Rough ER</div>
+    <!-- Include touch punch -->
+    <script type="text/javascript" src="js/jquery.ui.touch-punch.js"></script>
+    <script type="text/javascript" src="js/jquery.ui.touch-punch.min.js"></script>
+
+    <!-- drag and drop animation script -->
+    <script type="text/javascript" src="js/slide46.js"></script>
+
+    <div id="master-container">
```

Comment:
we have only an image which doesn't have any heading/text. Do you think we should add some heading?

Reply:
Yes. I should add a heading. Any suggestions based on the image?

Reply:
The other one has "Eukaryotic Cell Review". I don't know what this is?  (Animal Cell?) May be Animal Cell Review could be heading?